packages/repository/src/repositories/repository.ts

```diff
@@ -37,6 +38,22 @@ export interface ExecutableRepository<T extends Model> extends Repository<T> {
   ): Promise<AnyObject>;
 }
 
+export type TransactionalEntityRepository<
+  T extends Entity,
+  ID,
+  Relations extends object = {}
+> = TransactionalRepository<T> & EntityCrudRepository<T, ID>;
+export interface TransactionalRepository<T extends Entity>
```

Comment:
Please add an empty line to visually delimit the definition of `TransactionalEntityRepository` and `TransactionalRepository`.

Please add a tsdoc comment for each type.